crates/red_knot_python_semantic/resources/mdtest/narrow/conditionals_elif_else.md

```diff
@@ -0,0 +1,46 @@
+# Narrowing for conditionals with elif and else
+
+## Positive contributions become negative in elif-else blocks
+
+```py
+def int_instance() -> int: ...
```

Comment:
We should fill in the bodies of helper functions in tests so that they don't start failing in the future when we start checking that functions actually return what they say they return:

```suggestion
def int_instance() -> int:
    return 42
```